nebullvm/operations/optimizations/compilers/openvino.py

```diff
@@ -25,6 +30,132 @@
 from nebullvm.tools.transformations import MultiStageTransformation
 
 
+
+class TensorFlowOpenVINOCompiler(Compiler):
+    supported_ops = {
+        "cpu": [
+            None,
+            # QuantizationType.STATIC,
+            # QuantizationType.HALF,
+        ],
+        "gpu": [],
+    }
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(
+        self,
+        model: Model,
+        model_params: ModelParams,
+        temp_file_path: Path,
```

Comment:
what is this `temp_file_path`? Is it passed when the execute is called?

Reply:
removed it,because I can access the tmp directory from the `Compiler `parameter